_posts/en/newsletters/2020-11-11-newsletter.md

```diff
@@ -0,0 +1,140 @@
+  highly ranked [nodes][node rankings] to provide an incoming
+  channel for a period of 2,016 blocks (about two weeks).  Additional
+  contract lengths and other features are planned for the future.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Add MuHash3072 implementation][review club #19055-2] is a PR ([#19055][Bitcoin
+Core #19055]) by Fabian Jahr (incorporating code originally written by Pieter
+Wuille) that implements the [MuHash algorithm][muhash mailing list] in C++.
+MuHash is a rolling hash algorithm that can be used to calculate the hash
+digest of a set of objects and efficiently update that digest when items are
+added to or removed from the set. The algorithm could be used to calculate a
+digest of the complete UTXO set, which would be useful for database consistency
+checks or for a fast sync method such as [assumeutxo][topic assumeutxo].
+
+The high-level concepts for MuHash had been discussed in [a previous review
+club meeting][review club #19055], so this meeting focused specifically on the
+specification and implementation of the algorithm.
+
+{% include functions/details-list.md
+  q0="How much state is stored inside the MuHash3072 rolling hash object? How
+      much data is returned to a user requesting the set hash?"
+  a0="The MuHash3072 object stores 3072 bits (384 bytes) of internal state.
+      This state is hashed using a 256-bit hash function to return 256 bits (32
+      bytes) to the end user. An optimization is to store both a numerator (for the
+      added items) and a denominator (for the removed items), which would require
+      6144 bits (768 bytes) of internal state, but would reduce the number of costly
+      modulo operations required."
+  a0link="https://bitcoincore.reviews/19055-2#l-47"
+
+  q1="How can we test for membership in the MuHash set?"
+  a1="MuHash is not an accumulator, so there is no efficient way to test for set
+      membership. The only way would be to recalculate the MuHash digest from the
+      entire set of objects and then compare the digests."
+  a1link="https://bitcoincore.reviews/19055-2#l-131"
+
+  q2="What does the `#ifdef HAVE___INT128` code in the MuHash implementation do?"
+  a2="`#ifdef HAVE___INT128` is a C++ preprocessor directive. It ensures that code
+      contained within the `#ifdef` and `#endif` directives is only compiled on
+      platforms that support 128-bit integers. Since the MuHash code is
+      operating on numbers that are many times larger than built-in integer types
+      can handle, those numbers are broken down into multiple 'limbs', which are
+      combined to make the full number. Larger limbs means fewer operations and
+      therefore [better performance][muhash performance]. Support for 128 bit
```

Comment:
one last hyphen managed to get away (s/128 bit/128-bit/)

Reply:
Thanks! Fixed.